src/transaction/consensus_parameters.rs

```diff
@@ -0,0 +1,63 @@
+/// Consensus configurable parameters used for verifying transactions
+
+#[derive(Copy, Clone, Debug, PartialEq, Hash)]
+#[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
+pub struct ConsensusParameters {
+    /// Maximum contract size, in bytes.
+    pub contract_max_size: u64,
+    /// Maximum number of inputs.
+    pub max_inputs: u8,
```

Comment:
Since we're doing breaking changes anyways, can you go ahead and widen these to `u64` in line with https://github.com/FuelLabs/fuel-specs/issues/298?

Reply:
done